### PR TITLE
Add specific config examples across docs

### PR DESCRIPTION
**Pull request by jkowall**
## Description of the changes
First pass on adding example configs versus CLI across sampling and backends.

## How was this change tested?
make spellcheck
make develop

## Checklist
- [X] I have read https://github.com/jaegertracing/jaeger/blob/master/CONTRIBUTING_GUIDELINES.md
- [X] I have signed all commits

### CODE REVIEW

File: content/docs/next-release-v2/badger.md [hunk 1]
```diff
@@ -5,18 +5,7 @@ hasparent: true
 
 [Badger](https://github.com/dgraph-io/badger) is an embedded local storage, only available
 with **all-in-one** distribution. By default, it acts as ephemeral storage using a temporary file system.
```
Comment (yurishkuro) [inline content/docs/next-release-v2/badger.md:7]: bad wording here, "all-in-one distribution", "ephemeral" (file persistence is not ephemeral)

File: content/docs/next-release-v2/cassandra.md [hunk 2]
```diff
@@ -14,24 +14,8 @@ Cassandra also has the following officially supported resources available from t
 - [Kubernetes Operator](https://github.com/k8ssandra/cass-operator) from DataStax
 
 #### Configuration
```
Comment (yurishkuro) [inline content/docs/next-release-v2/cassandra.md:16]: this is the first header in the file, shouldn't it be L1 (`#`)?

**Review by yurishkuro — APPROVED** [review]

File: content/docs/next-release-v2/badger.md (hunk 1 above)
Comment (jkowall) [reply content/docs/next-release-v2/badger.md:7]: This was copied from existing docs, updated in latest push.

File: content/docs/next-release-v2/cassandra.md (hunk 2 above)
Comment (jkowall) [reply content/docs/next-release-v2/cassandra.md:16]: Sorry I don't understand the comment. Do you want a title for the file called Cassandra too?

Comment (yurishkuro) [reply content/docs/next-release-v2/cassandra.md:16]: `####` is level-4 header, a page should start with level-1 header `#`

**Pull request merged**